### PR TITLE
Add an intro section for Enable docs

### LINKED ISSUE #600: Documentation: Enable docs could use an intro section
The kiva portion of the docs has a nice introductory section that gives insight into the questions "What is Kiva?", "What does it do?", "Why should I care?", etc. just before going into a "Kiva Concepts Section".  Conversely, Enable just jumps right into "Enable Concepts".
A short paragraph or so intro section I think could be very useful and hopefully relatively simple/quick to add.

Ref:https://docs.enthought.com/enable/kiva.html

### PR DESCRIPTION
**Pull request by jwiggins**
Fixes #600

### CODE REVIEW

File: docs/source/enable/overview.rst [hunk 1]
```diff
@@ -1,258 +1,278 @@
...
-nested. Containers are responsible for event dispatch, draw dispatch, and
-layout. Containers override a lot of Component methods, so that they behave more
-like containers than plain components do.
+======
+Enable
+======
+
+Enable is a library which provides an interactive 2D canvas, upon which one can
+build `interactive plots <https://docs.enthought.com/chaco>`_ or other
+applications. It is analogous to the HTML5 ``<canvas>`` element and provides a
```
Comment (aaronayres35) [inline docs/source/enable/overview.rst:7]: maybe add a link to html5 canvas docs?

**Review by aaronayres35 — APPROVED** [review]
LGTM, thank you!

Just one minor comment, about adding a link to html5 cavnas docs.  It may not be necessary though, and if you prefer against it I'm fine if it is left out

**Comment by jwiggins** [comment]
I'm happy to link to the excellent MDN documentation. Done!

**Comment by jwiggins** [comment]
Thanks for the feedback

**Pull request merged**